astropy/io/ascii/cds.py

```diff
@@ -647,6 +650,32 @@ def write(self, lines):
 
                 to_pop.append(i)   # Delete original ``SkyCoord`` column.
 
+            # If the whole column is a `Time` or if it contains `Time` values,
+            # convert values to Modified Julian Dates.
+            elif isinstance(col, Time) or isinstance(col[0], Time):
+                # If column already has a description, then append to it.
+                name, description = 'MJD', 'Modified Julian Date'
+                if isinstance(col, Column):
+                    if col.description is not None:
+                        description = col.description + ' (Modified Julian Date)'
+                    if col.name is not None:
```

Comment:
Even though you are using `isinstance(col, Column)` and `col.name` and `col.description` can be directly accessed for a column, but would it be better to make this the "mixin-safe" `col.info.name` and so on. Don't think it is possible to have a `Quantity` column with `Time` values, but maybe there is some "mixin-specific" case that we are missing...

Reply:
I haven't looked up the `QTable`s and `Quantity` columns. For now, I suppose using the `if` statement suffices our purpose. I can work on `Quantity` with `Time` values and other `QTable` specific scenario in a separate PR later.